applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -890,21 +890,19 @@ static void cloud_event_handler(const struct nrf_cloud_evt *evt)
 	switch (evt->type) {
 	case NRF_CLOUD_EVT_TRANSPORT_CONNECTING:
 		LOG_DBG("NRF_CLOUD_EVT_TRANSPORT_CONNECTING");
-		if (evt->status != NRF_CLOUD_CONNECT_RES_SUCCESS) {
-			LOG_ERR("Failed to connect to nRF Cloud, status: %d",
-				(enum nrf_cloud_connect_result)evt->status);
-		}
 		break;
 	case NRF_CLOUD_EVT_TRANSPORT_CONNECTED:
 		LOG_INF("NRF_CLOUD_EVT_TRANSPORT_CONNECTED");
 		break;
+	case NRF_CLOUD_EVT_TRANSPORT_CONNECT_ERROR:
+		LOG_ERR("Failed to connect to nRF Cloud, status: %d", evt->status);
```

Comment:
I know this is copy paste but all other events just log the event name and possible status so I'd change this to be consistent.

Reply:
Yes, the event name should be there. Because the status is logged separately, maybe logging only the event name is sufficient here.

Reply:
I would actually combine these two logging lines unless there is a reason to leave it as is. Theoretically some test case etc would be dependent on the 2nd line but I doubt it. And even then, we should probably just change that test.

Reply:
So, you only want to log the event name and status? That's fine by me and would be consistent with the other events, but it would (maybe) make the error a bit harder to understand.

CI is running now and if it passes, I will not change this. In case tests fail (again), I can change the logging.

Getting the tests to pass seems to be ridiculously hard. CI has been ran 18 times for the PR and it has passed once. I haven't found any real problems, though, so it's just caused by instability of the tests.

Reply:
Ideally yes, log event name and status like with other events. Duplication log lines seems unnecessary.

Reply:
I can create a new PR to change the SLM logging as you requested.